app/models/review.rb

```diff
@@ -3,9 +3,9 @@ class Review
   attr_reader :distance
   SPACE_INTERVALS = [12.hours, 3.day, 1.week, 2.week, 1.month]
 
-  def initialize(args)
-    @answer = args[:answer].strip.mb_chars.downcase if args[:answer]
-    @card_id = args[:card_id]
+  def initialize(card_id,answer=nil)
```

Comment:
Space missing after comma.<br>Surrounding space missing in default value assignment.